code/modules/projectiles/guns/projectile/automatic.dm

```diff
@@ -78,6 +78,23 @@
 	mag_type = /obj/item/ammo_box/magazine/smgm9mm
 	origin_tech = "combat=4;materials=2"
 
+/obj/item/weapon/gun/projectile/automatic/gepard
+	name = "\improper Gepard SMG"
+	desc = "A cheaply made 7.62x25mm submachine gun, designated 'Gepard'. Has a threaded barrel for suppressors."

```

Comment:
Have you checked this actually can be surpressed? As you haven't set the supressor capable var that I can see.

Reply:
This can be, it defaults to one, and if it doesn't, then I am surprised as the screenshot is in game, and not a mock up.